__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require('supertest')
+const app = require("../db/app");
+const seed = require('../db/seeds/seed')
+const db = require("../db/connection")
+
+const devData = require('../db/data/test-data/index');
+
+
+beforeAll(() => {
+    return seed(devData)
+  });
+  
+  afterAll(() => {
+    db.end()
+  });
+
+
+describe("CORE: GET - /api/topics", () => {
+    test("200: Endpoint should contain all topic objects in correct format", () => {
+        return request(app)
+          .get("/api/topics")
+          .expect(200)
+          .then(({ body }) => {
```

Comment:
The topics array should be on an appropriate key on the body, see other comment on controller